tests/gds_kernel_latency.c

```diff
@@ -517,34 +515,32 @@ int pp_close_ctx(struct pingpong_context *ctx)
 
         if(ctx->exp_send_info == 1)
         {
+                
```

Comment:
please remove this whitespace change and the others below as they simply add noise